setup/docker.md

```diff
@@ -12,12 +12,27 @@ Coder for Docker works with the following platforms:
 
 - macOS 10.10+ with
   [Docker Desktop 20.10](https://www.docker.com/products/docker-desktop)
+
+  **Note**: If your computer uses an [Apple silicon] processor, you will need to
```

Comment:
@khorne3 reorganized this to avoid needing a heading, WDYT?

Reply:
@jawnsy looks good to me!